Docs/source/usage/parameters.rst

```diff
@@ -1919,6 +1919,22 @@ In-situ capabilities can be used by turning on Sensei or Ascent (provided they a
     Default is ``<diag_name>.fields_to_plot = Ex Ey Ez Bx By Bz jx jy jz``.
     Note that the fields are averaged on the cell centers before they are written to file.
 
+* ``<diag_name>.particle_fields_to_plot`` (list of `strings`, optional)
```

Comment:
We at the moment have particle-derived fields like `rho` and `rho_<species>` above in the `fields_to_plot` list.
Is that confusing and should they be moved down here?

I think we should also document here if the per-cell averages respect the particle shape or just do "NGP" (next-grid-point) deposition for the averaged quantity. As I understand it right now, if I were to deposit only the weight `w` with your new method, it would be a less-precise `rho_<species>`, because you do NGP deposition of quantities. Maybe let's add this implementation detail (NGP) in the last paragraph?

Reply:
The way things are currently, I think it makes sense to keep `rho*` in the `fields_to_plot` list. It's not possible to calculate a density from `particle_fields_to_plot`, since it calculates `sum(parser_fn * w) / sum(w)`, where the sums are over the particles of a given species within the cell. Based on @dpgrote's suggestions, I'm going to add an option to just calculate `sum(parser_fn * w)` but that will be in a different PR.

Good point on the NGP interpolation. I'll note that in the documentation.